lib/logger.js

```diff
@@ -8,15 +8,12 @@ function Logger() {
 	Object.keys(Logger.prototype).forEach(function (key) {
 		this[key] = this[key].bind(this);
 	}, this);
+
+	this.reporter = reporter;
```

Comment:
Yes, `Logger` passes `api` to reporters. You can set `api` for reporters in `cli.js` too, I think.